split-webapp/split/src/components/counter.js

```diff
@@ -1,43 +1,46 @@
-import React from 'react';
-import { increment } from '../redux/actions';
-import { connect } from 'react-redux';
+import React from "react";
```

Comment:
We should make a note to remove this file soon as it was part of the unused redux set up. Perhaps another PR regarding overall code clean up? I think it is fine to leave it in for now.

Reply:
Ah, I thought that redux was potentially going to be used in the future, so I left some of the template code. But yeah I think it'll be easy to remove them in a new PR for overall code clean up

Reply:
Jake and I were afraid of removing files unnecessarily, just in case we needed them in the future. But having said that, happy to remove these files to help with the project structure.